handsontable/src/3rdparty/walkontable/src/calculator/viewportRows.js

```diff
@@ -1,6 +1,8 @@
 import { ViewportBaseCalculator } from './viewportBase';
+import { StylesManager } from '../../../../stylesManager';
```

Comment:
I think that it's generally a bad idea to import the module to Walkontable outside its root directory. Leaving it like this may result in a more complex and/or longer refactor of the module - if one is ever done. The better option would be to pass the default cell size in the Walkontable settings (TableView class).